autoscale/src/main/java/com/sequenceiq/periscope/service/ha/LeaderElectionService.java

```diff
@@ -126,7 +114,6 @@ public void leaderElection() {
                     @Override
                     public void run() {
                         try {
-                            stackCollectorService.collectStackDetails();
```

Comment:
Nit: Remove the reference and import since the stackCollectorService isn't used here.

Reply:
Maybe also remove the entire StackCollectorService class if it is not used elsewhere.

Reply:
Removed the unused inject.
StackCollectorService and few other unused classes I will remove in a different PR.